source/_patterns/01-atoms/breadcrumb/index.js

```diff
@@ -2,15 +2,16 @@
  * Breadcrumb
  */
 
-// Module dependencies
-import base from 'base';
+// Custom
+import 'base';
 
 // Module styles
 import './_breadcrumb.scss';
 
-export default {
```

Comment:
I'm curious what are the advantages by swapping approaches here?

Reply:
All js files are "objects" from a module perspective. Our old way of doing this was essentially building an object in an "object", just to export. ES6 modules make this much simpler with the new structure.